pkg/thing/entities/config.go

```diff
@@ -2,9 +2,7 @@ package entities
 
 // Config represents the thing's config
 type Config struct {
-	SensorID       int         `json:"sensorId"`
-	Change         bool        `json:"change"`
-	TimeSec        int         `json:"timeSec,omitempty"`
-	LowerThreshold interface{} `json:"lowerThreshold,omitempty"`
-	UpperThreshold interface{} `json:"upperThreshold,omitempty"`
+	SensorID int    `json:"sensorId"`
+	Schema   Schema `json:"schema"`
+	Event    Event  `json:"event"`
```

Comment:
As `event` will be optional, set the json flag `omitempty`.

Reply:
Done!